puppet/modules/chassis/templates/wp-cli.local.yml.erb

```diff
@@ -0,0 +1,3 @@
+# This is generated automatically by Puppet. Edit at your own risk.
```

Comment:
Should note:
```suggestion
# This is generated automatically by Puppet. Rather than editing,
# override these settings in a wp-cli.local.yml file.
# See https://make.wordpress.org/cli/handbook/references/config/
```

Reply:
Doneskis.